attr_default.gemspec

```diff
@@ -18,6 +18,5 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = AttrDefault::VERSION
 
-  gem.add_dependency 'hobofields', '~> 0.9'
```

Comment:
Why was `hobofields` removed from the gemspec? Does this gem not use `hobofields`?

Reply:
We determined that in the end the gem doesn't actually use hobofields at all even though its mentioned. All tests pass when removing the gem as well so I got rid of it

Reply:
Ah, ok. That _says_ to me that we aren't actually testing this gem against hobofields. That's unfortunate; we should be testing this gem against hobofields since it has assumptions about how to work with hobofields. That said, this lack of test coverage is nothing new, so it's fine to remove the dependency for now!